Core2/Onboarding/src/main/java/com/infomaniak/library/onboarding/OnboardingScaffold.kt

```diff
@@ -147,6 +152,13 @@ private fun Preview() {
             ) {
                 Text("Bottom content")
             }
-        }
+        },
+        indicatorStyle = IndicatorStyle(
+            inactiveColor = Color.LightGray,
+            activeColor = Color.DarkGray,
+            inactiveSize = 8.dp,
+            activeWidth = 16.dp,
+            indicatorSpacing = 8.dp,
```

Comment:
Any reason to not use `Margin.Mini` here ?

Reply:
Yeah, they're not accessible because this is its own submodule that needs to be independent from SwissTransfer where the margins are defined. Ideally the margins definition should be moved to core altogether at some point probably